src/main/java/frc/robot/subsystems/Wrist.java

```diff
@@ -43,6 +44,15 @@ public Wrist(int WRIST_ID, float WRIST_HIGH_LIM, float WRIST_LOW_LIM) {
 
     wristNeo.enableSoftLimit(SoftLimitDirection.kForward, true);
     wristNeo.enableSoftLimit(SoftLimitDirection.kReverse, true);
+
+    wristNeo.setPeriodicFramePeriod(PeriodicFrame.kStatus0, 65535);
+    wristNeo.setPeriodicFramePeriod(PeriodicFrame.kStatus1, 65535);
+    wristNeo.setPeriodicFramePeriod(PeriodicFrame.kStatus2, 65535);
+    wristNeo.setPeriodicFramePeriod(PeriodicFrame.kStatus3, 65535);
+    wristNeo.setPeriodicFramePeriod(PeriodicFrame.kStatus4, 65535);
+    wristNeo.setPeriodicFramePeriod(PeriodicFrame.kStatus5, 65535);
```

Comment:
Status frame 5 has the absolute encoder data on it. Is that necessary for accessing the position in `isAtTarget`? Or does this work without it? I think that's the only status frame we _might_ need

Reply:
Yes, I updated that status rate back to the default.